src/docs/product/performance/transaction-summary.mdx

```diff
@@ -32,14 +32,16 @@ While you can use the search field to narrow down your events, you can also filt
 
 - **Duration Breakdown:** By graphing [P50](/product/performance/metrics/#p50-threshold), [P75](/product/performance/metrics/#p75-threshold), [P95](/product/performance/metrics/#p95-threshold), [P99](/product/performance/metrics/#p99-threshold), and P100 durations distinctly, you can utilize this display to see their transaction performance over time. Toggle the display of each segment by clicking on the legend (including Releases). Also, zoom in on specific slices to investigate spikes or possible performance regressions from a release.
 
-- **Latency Histogram:** Visualize the frequency of transactions that occur within each response time segment. This histogram view can help you understand the shape of transaction response times and quickly see outliers. Click on specific segments to drill down for more granularity.
-
 - **Duration Percentiles:** This graph shows the [average transaction duration](/product/performance/metrics/#average-transaction-duration) across distinct percentiles. For example, see how different the P50 and P99 response time durations are for a given transaction and understand the rate of increase between segments.
 
+- **Duration Distribution:** TBA
```

Comment:
```suggestion
- **Duration Distribution:** This graph shows the volume of transactions per median duration.
```

Reply:
@udameli I've updated the description, let me know if it makes sense to you!

Reply:
looks good! ty!